run_configs/jules/build_jules.py

```diff
@@ -70,8 +72,14 @@ def jules_config(revision=None):
             common_flags=[
                 '-c',
                 '-J', '$output'],
+            # required for newer compilers
+            # path_flags=[
+            #     AddFlags('*/io/dump/read_dump_mod.f90', ['-fallow-argument-mismatch']),
+            # ]
```

Comment:
I think, since this is not part of the fab source and just our idea of user config, we can get away with leaving comments if they have value. I feel this does have value because we need at least two versions of this config, including one for newer compilers which would need this bit. I guess it's a reminder that we'll need this for newer compilers, in a world where our environment has seemingly changed under our feet once already. Ideally, Fab (or the configs) would include a container spec so we could just focus on a single environment in these exploratory configs.